### PR TITLE
feat(accordion): add animation to accordion

### PR DESCRIPTION
**Pull request by ju-Skinner**
# Description

During the AppFrame design review, we agreed that we wanted to add a transition animation for the accordion

Loom video: https://www.loom.com/share/de94e5a22a8f41f39ec47e61699aef6f

Fixes #(issue)
https://kajabi.atlassian.net/browse/DSS-543

* Adds animation when opening the accordion
* Marker also animates when opening.

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [ ] tested manually
- [x] other:

# Checklist:

If not applicable, leave options unchecked.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

### CODE REVIEW

File: libs/core/src/components/pds-accordion/pds-accordion.scss [hunk 1]
```diff
@@ -64,3 +80,15 @@ summary {
...
+@keyframes fadeIn {
+ 0% {
+   opacity: 0;
+   transform: translateY(-10px);
+ }
+
+ 100% {
+   opacity: 1;
+   transform: translateY(0);
+ }
```
Comment (anechol) [inline libs/core/src/components/pds-accordion/pds-accordion.scss]: I'm realizing now as I see this that there isn't a fade out animation when the Accordion closes. Do you think that would be worth adding?

**Review by anechol — APPROVED** [review]
Nice job on adding this. Just one non-blocking comment.

**Review by QuintonJason — APPROVED** [review]

**Comment by ju-Skinner** [comment]
Updated loom video for commit [57c841f](https://github.com/Kajabi/pine/pull/154/commits/57c841f7806eaaf280523b37ae16412ea0495e9a)

https://www.loom.com/share/0dc0962e627840329401bdfecf8ece41

File: libs/core/src/components/pds-accordion/pds-accordion.scss (hunk 1 above)
Comment (ju-Skinner) [reply libs/core/src/components/pds-accordion/pds-accordion.scss]: @anechol We have moved away from the fading in or out. We are now just doing a simple transition that gives the appearance of sliding down.

File: libs/core/src/components/pds-accordion/pds-accordion.scss [hunk 2]
```diff
@@ -17,16 +18,38 @@ details {
...
+    }
+  }
+
+  /* stylelint-disable-next-line selector-no-qualifying-type */
+  .pds-accordion__body {
+    transition: max-height 1s ease-in-out;
+
+    /* stylelint-disable-next-line selector-no-qualifying-type */
+    &.open {
+      max-height: 5000px;
```
Comment (anechol) [inline libs/core/src/components/pds-accordion/pds-accordion.scss]: How do you feel about using `max-content` here instead of 5k?

**Review by anechol — APPROVED** [review]
Nice work on this. Just one non-blocking comment.

**Review by pixelflips — APPROVED** [review]
Nice updates here to provide some delightful UX! LGTM! 👍🏼

File: libs/core/src/components/pds-accordion/pds-accordion.scss (hunk 2 above)
Comment (ju-Skinner) [reply libs/core/src/components/pds-accordion/pds-accordion.scss]: @anechol see commit [2e0f1e5](https://github.com/Kajabi/pine/pull/154/commits/2e0f1e546fc490de2e13fdfabc3aafe38df65dc6) for change

**Review by anechol — APPROVED** [review]
🚀 Nice work!

**Pull request merged**